src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 1120px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+@media(max-width: 1024px) {
+  .container {
+    max-width: 944px;
+  }
+}
+
+.nav__logo {
+  padding: 10px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__link {
+  display: block;
+  font-family: "Roboto", serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  padding: 23px 10px;
```

Comment:
Horizontal padding might be useful here (extend clickable zone), but that's the reason tests fail (use horizontal margins to fix tests)